src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,17 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  value: [],
+  status: 'Under construction',
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+
+  },
+});
```

Comment:
- [ ] Kindly modify the `createSlice` by adding the appropriate `checkStatus` action. The "under construction" text should be returned from the reducer, not the initial state as you have done here.